ertviz/controllers/plot_view_controller.py

```diff
@@ -0,0 +1,88 @@
+# Could go into seperate file
```

Comment:
I guess this comment can be removed now?

Reply:
👍